integ/scripts/bash/run-e2e-automated.sh

```diff
@@ -32,7 +32,7 @@ refreshcreds() {
 
 # Basic integ test configuration
 export SKIP_TEST_CONFIG=true
-export DEADLINE_VERSION="10.1.11.5"
+export DEADLINE_VERSION
```

Comment:
I do not like when such variable is hidden in comments.
What do you say if we left it like in `test-config.sh` with the same comment?

Reply:
I don't think we should leave it like `test-config.sh` because the two scripts serve different purposes.

`test-config.sh` is meant to be a config file that someone would manually edit before manually kicking off the integration tests.

`run-e2e-automated.sh` is meant to be the entry point for the tests when running them in an automated fashion. It gets executed from our pipeline, where you don't have the opportunity to manually edit this value before the tests get executed, but we do have the opportunity to configure environment variables to be set before the execution.

If we remove this line, we could set the `DEADLINE_VESION` environment variable before running the script, if we leave this line here, it will reset the env var set by the user, forcing the tests to always use the latest version.